extensions/sql-database-projects/src/test/projectController.test.ts

```diff
@@ -597,7 +598,7 @@ describe('ProjectsController', function (): void {
 			should(opened).equal(true);
 		});
 
-		it.skip('Callbacks are hooked up and called from create project from database dialog', async function (): Promise<void> {
+		it('Callbacks are hooked up and called from create project from database dialog', async function (): Promise<void> {
```

Comment:
this test was still failing in the PR validation

Reply:
I'm going to leave this one as skipped, and fix it separately.  I'm hitting something weird with the UI framework.